### PR TITLE
Support#499 Docker npm node mismatch

### PR DESCRIPTION
**Pull request by BrianXu20**
Ticket: https://projects.torchbox.com/projects/support-team/tickets/499
Docker node version: 16
Project node version: 12
Screen Shot 2022-06-08 at 8 44 10 PM

### CODE REVIEW

File: rca/static_src/javascript/programmes/components/ProgrammesCategories/CategoriesPanels.js [hunk 1]
```diff
@@ -18,7 +18,6 @@ const CategoriesPanels = ({ categories, activeCategory }) => {
                     key={c.id}
                     id={c.id}
                     className="categories-panels__panel"
-                    role="tabpanel"
```
Comment (BrianXu20) [inline rca/static_src/javascript/programmes/components/ProgrammesCategories/CategoriesPanels.js:21]: Fix this issue
https://app.circleci.com/pipelines/github/torchbox/rca-wagtail-2019/5435/workflows/12ed1ee6-7321-428d-a1ab-e110cf53753d/jobs/6993
```
/Users/bxu/Documents/rca-wagtail-2019/rca/static_src/javascript/programmes/components/ProgrammesCategories/CategoriesPanels.js
  17:17  error  The attribute aria-expanded is not supported by the role tabpanel  jsx-a11y/role-supports-aria-props
```

Comment (BrianXu20) [reply rca/static_src/javascript/programmes/components/ProgrammesCategories/CategoriesPanels.js:21]: take out role should have a minimum impact on the FE

**Review by BrianXu20 — COMMENTED** [review]
added some comments for reviewer

**Review by nickmoreton — APPROVED** [review]
Happy to approve ~~but I'd be interested about removing the `role="tabpanel"`~~

**Pull request merged**